aggregator/.env.test

```diff
@@ -23,7 +23,7 @@ MAX_UNCONFIRMED_AGGREGATIONS=3
 LOG_QUERIES=true
 TEST_LOGGING=true
 
-REQUIRE_FEES=true
+REQUIRE_FEES=false
```

Comment:
The provider does not yet support paying the aggregator. I'll be making these changes next, so will be setting this back to true once that is done. Thought making that change would be out of scope for this PR.